dali/pipeline/operator/op_schema.cc

```diff
@@ -129,6 +134,10 @@ a pipeline scope. False if it was defined without pipeline being set as current.
                  "Operator name as presented in the API it was instantiated in (without the module "
                  "path), for example: cast_like or CastLike.",
                  OperatorName());
+
+  DeprecateArg("seed", true,
+               "The argument \"seed\" should not be used with operators that don't use "
+               "random numbers.");
```

Comment:
nitpick: In runtime, great, we don't want to break stuff. In the docs, I wonder if we would be better off just without any reference to the seed args for non-randomized ops. Now it renders as big orange warning in the list of the args.

Reply:
Great point - I'll try to hide it somehow.

Reply:
Done.